### DISCUSSION TITLE
Backfill: `Missing required config entry`

### DISCUSSION THREAD

**Discussion by the4thamigo-uk** [dagster-io/dagster#18730] [Community]
Hi I have a job running two partitioned assets A and B such that B depends on A. I have some partitions that have failed with a transitory error. When I try to backfill the failed partition using the UI, I get the following error : 

```
Missing required config entry "ops" at the root. Sample config for missing entry ...
```
However, if I go the the run that failed and hit the `Re-execute` button, the run executes.

How do I fix this so I can successfully  reprocess the partition using a backfill?

**Comment by sryza** [comment]
Hi @the4thamigo-uk - presumably you're hitting this error because one of your assets requires config? Are you backfilling via the job page, and does your job use PartitionedConfig?

**Reply by the4thamigo-uk** [reply]
Right, I think I understand. My sensor was setting the config for the job in order to pass the name of a file that it had detected in external s3 bucket. Obviously when I ran a backfill the config wasn't set. I have restructured the code so it doesn't use config in this way, although it does require me to have logic to rediscover the file again in the asset.

**Comment by hedeesaa** [comment]
I'm facing the same issue. How can we pass config to a backfilling job? 
I checked the LaunchPartitionBackfill graphql query, it doesn't have accept run_config.
